tests/view/crudapi/test_crud_api.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import werkzeug
+
+from http import HTTPStatus
+from timeless.views import CrudAPIView
+from timeless.restaurants.models import Location
+"""
+    Tests for CrudeAPIView.
+    
+    @todo #221:30min Continue with the implementation of CrudAPIView.
+     Implement post, put and delete methods. We should return json
+     representation of object model in methods. After that remove the ignore annotation from tests on
+     test_crud_api.py.
+"""
+
+
+@pytest.mark.skip
+def test_get_found_object():
+    expected = Location(
+        name="Los Pollos Hermanos Flagship Restaurant",
+        code="FR",
+        company_id=1,
+        country="United States",
+        region="Southwest",
+        city="Albuquerque",
+        address="1200 - 12100 Coors Rd SW",
+        longitude=35.1046478,
+        latitude=-106.6885328,
+        type="T",
+        status="open",
+        comment="Flagship restaurant of Los Pollos Hermanos, from a famous television series"
+    )
+    apiview = CrudAPIView(model=Location, url_lookup="location_id")
+    result = apiview.get(apiview,location_id=5)
```

Comment:
@paulodamaso I guess here we have the same problem that in #228

Reply:
@emilianodellacasa @paulodamaso yes, we have :) Also we cannot pass args to constructor, since it has to constructor. Flask will make a function from this class, see https://github.com/pallets/flask/blob/master/flask/views.py#L77

For `LocationAPIView` it will work the following way:
```python
class LocationAPIView(CrudAPIView):
    model = Location
    url_lookup = "locaton_id"
```

So if you want to emulate `Location` behavior, you have to do it as you did before review fixes: 
```
    apiview = CrudAPIView()
    apiview.model = Location
    apiview.url_lookup = "location_id"
```